pitch.md

```diff
@@ -23,38 +79,101 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+communicated to the user through a console of sorts.
+
+
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to set up a simple program using one of the "higher LED actions." It should also be easy to set its color, and display time. It should
+be possible to chain together these actions into more complex compositions.
+Also, it should be possible but difficult to create other "higher LED actions"
+out of the lower level actions. For now, it should be impossible or very
+difficult to control single LEDs and to use any sort of control flow while
+creating the effects. 
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+The only information I could find is that to control these lights you must use 
+Arduino or Raspberry Pi code to control the wire outputs for the LEDs. I would 
+not consider thisa  DSL in my domain but rather a DSL in the domain of all 
+electrical engineering projects. I think the reason why there hasnt been a DSL 
+is either because those interested in the domain are okay using the regular 
+Arduino code or because it is quite a specific domain so it doesnt seem likely 
+to have languages created for it. 
+
+Here is a link to the arduino library for addressable LEDs 
+https://learn.adafruit.com/adafruit-neopixel-uberguide/arduino-library 
+which really shows the complexity of the language. 
+
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I think that a lot (most maybe) of this project will be working on the **
+language** aspects. Once the back end methods of creating a fade function/color 
+change function/etc.. are completed, the rest will be trying to design the 
+language in a way that promotes chaining commands and easily integrates aspects 
+of looping and time to allow for complex light effects. I have already come up 
+with a few options but picking the right one is difficult because there are 
+lots of trade-offs to think of. 
+
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project could be quite ambitious. There are lots of features that could be 
+made for the user and it is easy to get lost in "feature creating land" instead 
+of language design land. However, I also think the scope is not too large since 
+it focuses on a relatively specific domain. 
```

Comment:
It could be ambitious, but I think it is possible to start really small and get something that just works. Then you can build it up as much as you want.